src/index.ts

```diff
@@ -83,19 +82,15 @@ class SOOSSCAAnalysis {
       },
     );
 
-    analysisArgumentParser.argumentParser.add_argument("--packageManagers", {
-      help: "A list of package managers, delimited by comma, to include when searching for manifest files.",
-      required: false,
-      default: [],
-      type: (value: string) => {
-        if (value.length === 0) return [];
-        const values = value.split(",");
-        values.map((value) => {
-          return ensureEnumValue(PackageManagerType, value, "--packageManagers");
-        });
-        return values;
+    analysisArgumentParser.addEnumArgument(
+      analysisArgumentParser.argumentParser,
+      "--packageManagers",
+      {
+        help: "A list of package managers, delimited by comma, to include when searching for manifest files.",
+        required: false,
+        default: [],
```

Comment:
when package managers isnt in the command it's undefined
when it's specified as --packageManagers="" it takes the default `[]`
If we dont have a default it does the ensure enum value method on the api client and says that "" isn't a valid option